README.md

```diff
@@ -27,18 +27,21 @@ License Notes
 -------------
 
 The project code is licensed under the terms of [MIT license][license]. The
-original resourses from JMathTeX (`DefaultTexFont.xml`, `GlueSettings.xml`,
-`PredefinedTexFormulas.xml`, `TexFormulaSettings.xml`, `TexSymbols.xml`) are
-redistributed under the terms of [GNU GPL v2][gpl].
+original resourses from [JMathTeX][jmathtex] (`DefaultTexFont.xml`,
+`GlueSettings.xml`, `PredefinedTexFormulas.xml`, `TexFormulaSettings.xml`,
+`TexSymbols.xml`) are taken from the [GPLv2-distributed][gpl]
+[JMathTeX][jmathtex], but JMathTeX authors have granted a permission to
+redistribute these resourses under the MIT license.
```

Comment:
Well, I have some comments on that.

1. The wrap should be removed for now because it contradicts with other parts of the document; I'll do that.
2. I prefer to include wrapping into all of markdown files I'm authoring. I'm generally trying to follow old [Markdown Style Guide](http://tirania.org/blog/archive/2014/Sep.html) by Miguel de Icaza, because it was the only guide I've found long ago.

   These days, we have modern markdown linters and some additional style guides published, but I haven't checked them.

   Both wrapped and not wrapped documents have their pros and cons, so I think we should in future check some style guides and enable markdown linter. For example, nodejs guys author *a lot* of markdown and [have a style guide](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md#style-guide) probably best suited for collaborative work on documentation.